packages/koa-shopify-graphql-proxy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
+## [3.4.0] - 2020-04-15
```

Comment:
The version and the date need to be added by the person releasing the new version. You need to add your changelog entry under `## [Unreleased]`. Simply uncomment line 8.